### PR TITLE
[Suggestion] Update reviewprocess docs for reflect eng docs changes

### PR DESCRIPTION
**Pull request by FacerAin**
### Requests
- https://github.com/Azure/azure-sdk/commit/c03f8f2c268e3bcabdef2d36b655872dfe2c7a70 There are some changes in eng review_process docs which add tables and sentences.
- For readability, I would like to incorporate their changes into the Korean docs.

kor: https://azure.github.io/azure-sdk-korean/policies_reviewprocess.html
eng: https://azure.github.io/azure-sdk/policies_reviewprocess.html

### Related Issue
close: #150

### CODE REVIEW

File: docs/policies/reviewprocess.md [hunk 1]
```diff
@@ -7,134 +7,90 @@ sidebar: general_sidebar
...
 
-그 외 모든 언어에 대해서는 아키텍처 보드에 요청하여 개별적으로 가장 적합한 형식을 논의하십시오.
+**준비물**:
+|제목 | 중요도 | 간단한 설명 | 예시 및 지원 문서 |
+|--|--|--|--|
+| 주요 시나리오 | 필수 | 서비스가 활용되는 주요 시나리오 들| 주요 시나리오를 식별하는 방법에 대한 가이드라인 - [링크](https://github.com/Azure/azure-sdk-pr/blob/24384df0202021ab86ee37fcb14e9554182cd014/training/azure-sdk-apis/principles/approachable/README.md#hero-scenarios)<br><br> [예시](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample1_DetectLanguage.md) |
+| 핵심 개념 | 권장 | 명사 및 동사의 용어집 | [예시](https://github.com/Azure/azure-sdk-pr/blob/main/onboarding/Core_Concepts.pdf) |
+| APIView | 권장 | 생성된 SDK용 APIView | [예시](https://apiview.dev/Assemblies/Review/8b7f5312697a458ab9e65c2fd9cdc2dd)  |
+| REST API 스펙 | 권장 | [azure/azure-rest-api-specs-pr](https://github.com/azure/azure-rest-api-specs-pr) 또는 [azure/azure-rest-api-specs](https://github.com/azure/azure-rest-api-specs) 저장소의 검토된 REST API 사양 정의에 대한 링크 | [예시](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/attestation/data-plane/Microsoft.Attestation/stable/2020-10-01/attestation.json) |
```
Comment (FacerAin) [inline docs/policies/reviewprocess.md:55]: https://github.com/azure/azure-rest-api-specs-pr
I am unable to access the link provided above. Is it a private repository for an internal team, or is the link invalid?

**Comment by FacerAin** [comment]
> @microsoft-github-policy-service agree [company="{your company}"]

@microsoft-github-policy-service agree [company="{Kyunghee University}"]

**Comment by FacerAin** [comment]
> @FacerAin please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by FacerAin** [comment]
> @microsoft-github-policy-service agree

@microsoft-github-policy-service agree

File: docs/policies/reviewprocess.md (hunk 1 above)
Comment (ianychoi) [reply docs/policies/reviewprocess.md:55]: Thank you - how about mentioning like `[azure/azure-rest-api-specs-pr (직원 액세스)](https://github.com/azure/azure-rest-api-specs-pr)` (source: https://github.com/Azure/azure-rest-api-specs/wiki )?

**Review by ianychoi — CHANGES_REQUESTED** [review]
Great - would you reflect the change?

**Comment by FacerAin** [comment]
@ianychoi 
I reflected the change you mentioned. Thank you for review☺️

**Pull request merged**